### PR TITLE
[SW-19] Receive popup

### PR DESCRIPTION
**Pull request by subhod-i**

### CODE REVIEW

File: src/popup/router/components/AssetSelector.vue [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    },
+    tokenList() {
+      return [
+        ...(this.aeternityToken ? [this.aeternityToken] : []),
+        ...this.convertedTokenInfo,
+      ];
+    },
+  },
+  created() {
+    this.$emit('change', JSON.parse(JSON.stringify(this.currentToken)));
```
Comment (subhod-i) [inline src/popup/router/components/AssetSelector.vue:69]: This is(parse and Stringify) needed to match the default option value.

**Review by pstachel — CHANGES_REQUESTED** [review]

File: src/popup/router/components/Modals/TransferReceive.vue [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+          data-cy="account-name"
+          class="account-name"
+        >
+          {{ $t('pages.account.heading') }} {{ accountIdx + 1 }}
+        </span>
+      </div>
+
+      <div class="address-info">
+        <div class="qrcode-cover">
+          <qrcode-vue
```
Comment (peronczyk) [inline src/popup/router/components/Modals/TransferReceive.vue:32]: I suggest to use PascalCase.

File: src/popup/router/components/RequestField.vue [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+    >
+      <label
+        v-if="label || $slots.label"
+        class="label"
+      >
+        {{ label }}
+        <slot name="label" />
+      </label>
+      <div class="main-wrapper">
+        <main
```
Comment (peronczyk) [inline src/popup/router/components/RequestField.vue:19]: You can use only one main on whole page.

File: src/popup/router/routes/index.js [hunk 4]
```diff
@@ -35,7 +35,7 @@ import Retip from '../pages/Retip.vue';
 import SecuritySettings from '../pages/SecuritySettings.vue';
 import Transfer from '../pages/Transfer.vue';
 import TransferSend from '../pages/TransferSend.vue';
-import TransferReceive from '../pages/TransferReceive.vue';
+import TransferReceive from '../components/Modals/TransferReceive.vue';
```
Comment (peronczyk) [inline src/popup/router/routes/index.js:38]: Using a modal page as a route component will break the possibility of closing the modal. This component needs to be wrapped with another one that will manage the open state and react on `close` event when user clicks "X" or somewhere around the modal.

File: src/popup/router/components/Modals/TransferReceive.vue (hunk 2 above)
Comment (subhod-i) [reply src/popup/router/components/Modals/TransferReceive.vue:32]: For the css class?

File: src/popup/router/components/Modals/TransferReceive.vue [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+
+    .account-info {
+      margin-top: 4px;
+      display: flex;
+      justify-content: center;
+      align-content: center;
+
+      .avatar {
+        width: 16.62px;
+        height: 16.62px;
```
[the remark this answers is not in the document]
Comment (subhod-i) [reply src/popup/router/components/Modals/TransferReceive.vue:225]: Using at the changes I made

File: src/popup/router/components/Modals/TransferReceive.vue [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+    width: 360px;
+    overflow: hidden;
+    overflow-y: auto;
+  }
+
+  .transfer-receive {
+    display: flex;
+    align-content: center;
+    justify-content: center;
+    flex-direction: column;
```
[the remark this answers is not in the document]
Comment (subhod-i) [reply src/popup/router/components/Modals/TransferReceive.vue:200]: Using at the changes I made

File: src/popup/router/components/Modals/TransferReceive.vue (hunk 2 above)
Comment (subhod-i) [reply src/popup/router/components/Modals/TransferReceive.vue:32]: If qrcode-vue tag then I am using the existing one and I guess the library supplies tag in the same manner

Comment (peronczyk) [reply src/popup/router/components/Modals/TransferReceive.vue:32]: Yes, I'm refering to the component name. We are using PascalCase everywhere and here the name also should follow the rule ("QrcodeVue").

**Comment by subhod-i** [comment]
@peronczyk Closed?

**Pull request closed without merging**